src/main/java/list_Entry_Todo.java

```diff
@@ -0,0 +1,11 @@
+public class list_Entry_Todo extends list_Entry {
+
+    public list_Entry_Todo(String task, boolean check) {
```

Comment:
May be good to add more JavaDocs to explain the organisation and purpose of the different methods/classes?